test/CAPBookshop.test.js

```diff
@@ -0,0 +1,45 @@
+const { wdi5 } = require('wdio-ui5-service');
+const lib = require('wdi5-fe-selectors');
+
+describe('samples', () => {
+
+    it('should log', () => {
+        const logger = wdi5.getLogger();
+        logger.log('hello world!');
+    })
+
+    it('go home', async () => {
+        await lib.goHome();
+        await _sleep(2 * 1000)
```

Comment:
not sure why this is actually required - I get that FE is slow in general (aka resource-intensive), but waiting 2 secs is just too long IMO. 
Best try to get rid of any manual wait operations and leave it to the underlying framework such as `wdi5` to sync app- and test-runtime

Reply:
removed as it is not needed